app/src/main/java/com/internship/move/feature/onboarding/model/ViewPagerItem.kt

```diff
@@ -0,0 +1,9 @@
+package com.internship.move.feature.onboarding.model
+
+data class ViewPagerItem(
+    val id: Int,
+    val photoUrl : Int,
```

Comment:
The naming here is a bit misleading, photos are resources in this case, not urls